recipes/tiny-dnn/all/conanfile.py

```diff
@@ -0,0 +1,67 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class TinyDnnConan(ConanFile):
+    name = "tiny-dnn"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/tiny-dnn/tiny-dnn"
+    description = "tiny-dnn is a C++14 implementation of deep learning."
+    topics = ("header-only", "deep-learning", "embedded", "iot", "computational")
+    settings = "compiler", "os"
+    requires = "cereal/1.3.0"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
```

Comment:
tiny-dnn has many options.

https://github.com/tiny-dnn/tiny-dnn/blob/v1.0.0a3/CMakeLists.txt#L47-L62

I think it would be better to explicitly set the options not needed (e.g. `BUILD_TESTS`...), and set the ones which are not yet supported with TODO to make them available in the future (e.g. `USE_SSE`...)

Reply:
Indeed it offers many options, I tried to keep it simple as possible, I don't know how popular is it and how deep we should work for this recipe. It's a port from Conan Community actually.

Reply:
Ok, if someone needs something, they will still have to make the PR, so let's keep it simple.

Reply:
_Note for someone, who will want to change the recipe_:
Actually, there are some options, which do not depend on external libraries (`USE_SSE`, `USE_AVX `, `USE_AVX2 `, `USE_SERIALIZER`, `USE_DOUBLE`)

Reply:
let's go as-is and add options on demand